engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -2645,6 +2647,27 @@ typedef struct {
   int64_t engine_id;
 } FlutterProjectArgs;
 
+typedef struct {
+  /// The size of this struct. Must be
+  /// sizeof(FlutterDispatchSemanticsActionInfo).
+  size_t struct_size;
+
+  /// The ID of the view that includes the node.
+  FlutterViewId view_id;
+
+  /// The semantics node identifier.
+  uint64_t node_id;
+
+  /// The semantics action.
+  FlutterSemanticsAction action;
+
+  /// Data associated with the action.
+  const uint8_t* data;
+
+  /// The data length.
+  size_t data_length;
+} FlutterDispatchSemanticsActionInfo;
```

Comment:
Let's update this struct to match the `FlutterEngineSendSemanticsAction` function name:

```suggestion
typedef struct {
  /// The size of this struct. Must be
  /// sizeof(FlutterSendSemanticsActionInfo).
  size_t struct_size;

  /// The ID of the view that includes the node.
  FlutterViewId view_id;

  /// The semantics node identifier.
  uint64_t node_id;

  /// The semantics action.
  FlutterSemanticsAction action;

  /// Data associated with the action.
  const uint8_t* data;

  /// The data length.
  size_t data_length;
} FlutterSendSemanticsActionInfo;
```